src/vue/pages/sales/SalesList.vue

```diff
@@ -149,8 +152,38 @@ export default {
           })
       }
     },
+  },
+
+  watch: {
+    'filters.state' () {
+      this.saleUpdateKey++
+    },
+    isUserSales () {
```

Comment:
In which case can the value of a variable change?

Reply:
When going to `My sales` page from `Sales` page or vice versa